proto/qdrant/cloud/quota/v1/quota.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package qdrant.cloud.quota.v1;
+
+import "buf/validate/validate.proto";
+import "google/api/annotations.proto";
+import "qdrant/cloud/common/v1/common.proto";
+
+// Quota is the API used to configure Quotas.
+service QuotaService {
+  // Lists all quotas for the authenticated user.
+  // Required permissions:
+  // - None (authenticated only)
+  rpc ListAuthenticatedUserQuotas(ListAuthenticatedUserQuotasRequest) returns (ListAuthenticatedUserQuotasResponse) {
```

Comment:
Imo it should be GetX not ListX, in a list we expect a repeated X, which isn't the case in this API.